internal/nginx/extensions.go

```diff
@@ -17,6 +17,10 @@ func ParseLBMethod(method string) (string, error) {
 		method, err := validateHashLBMethod(method)
 		return method, err
 	}
+	if strings.HasPrefix(method, "random") {
```

Comment:
I suggest creating a map with valid input for NGINX OSS, similar to `nginxPlusLBValidInput`. It will include `least_conn`, `ip_hash` and combinations for the random method.